js/api_spec/witness_pb.js

```diff
@@ -0,0 +1,193 @@
+// source: witness.proto
+/**
+ * @fileoverview
+ * @enhanceable
+ * @suppress {missingRequire} reports error on implicit type usages.
+ * @suppress {messageConventions} JS Compiler reports an error if a variable or
+ *     field starts with 'MSG_' and isn't a translatable message.
+ * @public
+ */
+// GENERATED CODE -- DO NOT EDIT!
+/* eslint-disable */
+// @ts-nocheck
+
+var jspb = require('google-protobuf');
+var goog = jspb;
+var global = Function('return this')();
```

Comment:
Note: This use of eval (which is pretty critical to how they handle exports/code sharing) prevents the use of this generated code in chrome extensions and…most modern javascript environments.

Looking for alternatives now.